pkg/api/helper/helpers.go

```diff
@@ -14,6 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// NOTE: DO NOT use those helper functions through client-go, the
```

Comment:
We don't need to add the NOTE here now, we restrict the visibility of the package in the BUILD file.